doc/user-getting-started-docker-dotnet.md

```diff
@@ -182,6 +182,6 @@ To run an application with Mystikos in release or production mode, please see
 
 For more complex dotnet programs that are already working with Mystikos, please see:
 
-* [A web service based on ASP.net](https://github.com/deislabs/mystikos/tree/main/solutions/aspnet)
+* [A web service based on ASP.net](https://github.com/deislabs/mystikos/tree/main/solutions/aspnet_core)
 * [A web client that queries a remote attestation service](https://github.com/deislabs/mystikos/tree/main/solutions/dotnet/HelloWorld)
-* [An example written with Azure SDK for dotnet](https://github.com/deislabs/mystikos/tree/main/solutions/dotnet_azure_sdk)
\ No newline at end of file
+* [An example written with Azure SDK for dotnet](https://github.com/deislabs/mystikos/tree/main/solutions/dotnet_azure_sdk)
```

Comment:
I have been staring at this for a long time. What's the delta from the previous line 187?

Reply:
187 should be unchanged, I did the modification in the browser, not sure if it added a special space or character. Would you advice how I remove the change to 187, would I revert the change and squash the commit history ? thanks!

Reply:
Yes, please.